process/smartContract/process_test.go

```diff
@@ -2342,17 +2341,17 @@ func TestScProcessor_ProcessSmartContractResultWithData(t *testing.T) {
 	test := "test"
 	result := ""
 	sep := "@"
-	result = result + test
-	result = result + sep
-	result = result + test
-	result = result + sep
-	result = result + test
-	result = result + sep
-	result = result + test
-	result = result + sep
-	result = result + test
-	result = result + sep
-	result = result + test
+	result += test
```

Comment:
maybe you can refactor this sequence with a for loop

Reply:
done